ansible/roles_ocp_workloads/ocp4_workload_windows_node/defaults/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+become_override: False
+ocp_username: opentlc-mgr
+silent: False
+
+# Defaults values below are for OpenShift Windows Containers - Tech Preview
+ocp4_workload_windows_node_defaults:
```

Comment:
Hm. That's the "old" (as in tried it, didn't work) way of doing things. Can you please change to role name prefixes?

e.g. ocp4_workload_windows_node_namespace, ocp4_workload_windows_node_version, ...

Reply:
updated by #4fee590